src/styles/blocks/card.scss

```diff
@@ -0,0 +1,119 @@
+.card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+
+  font-size: 12px;
+
+  color: $main-card-color;
+
+  transition:
+    transform 300ms linear,
+    color 300ms linear;
+
+  &:hover {
+    cursor: pointer;
+    transform: scale(120%);
+  }
+
+  &__title {
+    display: block;
+    margin: 0 0 4px;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-decoration: none;
+
+    color: inherit;
+  }
+
+  &:hover &__title {
+    color: $card-title-hovered-color;
+  }
+
+  &__photo {
+    width: $card-photo-width;
+    height: $card-photo-height;
+
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__product-code {
+    display: block;
+    margin-bottom: 16px;
+
+    font-size: 10px;
+    line-height: 14px;
+
+    color: $secondary-card-color;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__stars {
```

Comment:
but why?
u already have a separate `stars.scss` file for stars and `stars` class for it
use it. Don't create a new one

Reply:
@VolodymyrKirichenko I did it to avoid setting position properties for block, as BEM says
Or if needed to reset I can do it?

Reply:
@Valeraom you use the stars block according to the BEM.
This is a separate block that can be used in any part of the code.
additional class `card__stars` is redundant